src/main/java/com/kodilla/ecommercee/mapper/OrderMapper.java

```diff
@@ -13,19 +11,26 @@
     public OrderEntity mapToOrder (final OrderDto orderDto) {
         return new OrderEntity(
                 orderDto.getId(),
-                orderDto.getProductGroup()
+                orderDto.getOrderNumber(),
+                orderDto.getProductGroup(),
+                orderDto.getPrice(),
+                orderDto.getProductList(),
+                orderDto.getUser()
         );
     }
     public OrderDto mapToOrderDto (final OrderEntity orderEntity) {
         return new OrderDto(
                 orderEntity.getId(),
-                orderEntity.getProductGroup()
+                orderEntity.getOrderNumber(),
+                orderEntity.getProductGroup(),
+                orderEntity.getPrice(),
+                orderEntity.getProductList(),
+                orderEntity.getUser()
         );
     }
     public List<OrderDto> mapToOrderDtoList(final List<OrderEntity> orderEntityList) {
-        System.out.println(orderEntityList);
         return orderEntityList.stream()
-                .map(t->new OrderDto(t.getId(), t.getProductGroup()))
+                .map(t->new OrderDto(t.getId(), t.getOrderNumber(), t.getProductGroup(), t.getPrice(), t.getProductList(), t.getUser()))
```

Comment:
mamy mappery żeby nie pisać tych konstruktorów, nie można ich tu użyć ?

Reply:
No pewnie, że tak. Zmieniłem na this::mapToProductDto